### ISSUE TITLE
Cannot install Windows Terminal Preview 1.15 as Midi dependency missing

### ISSUE THREAD

**Issue by glawrence** [microsoft/terminal#13520]
### Windows Terminal version

1.15.1862.0

### Windows build number

10.0.19041.1826

### Other Software

Windows Sandbox 10.0.19041.985

### Steps to reproduce

- Load Windows Sandbox
- Download latest Windows Terminal Preview build
- Download Depenedency Microsoft.VCLibs.x64.14.00.Desktop.appx from https://docs.microsoft.com/en-us/troubleshoot/developer/visualstudio/cpp/libraries/c-runtime-packages-desktop-bridge
- Open Windows PowerShell (5.1)
- Execute the following: `Add-AppXPackage .\Microsoft.WindowsTerminalPreview_Win10_1.15.1862.0_8wekyb3d8bbwe.msixbundle -DependencyPath .\Microsoft.VCLibs.x64.14.00.Desktop.appx`

### Expected Behavior

A successful install without an errors

### Actual Behavior

This is what happens when I execute the install command:

```PS C:\Users\WDAGUtilityAccount\Desktop\Sandbox> Add-AppXPackage ..\..\Downloads\Microsoft.WindowsTerminalPreview_Win10_1.15.1862.0_8wekyb3d8bbwe.msixbundle
Add-AppXPackage : Deployment failed with HRESULT: 0x80073CF3, Package failed updates, dependency or conflict validation.
Windows cannot install package Microsoft.WindowsTerminalPreview_1.15.1862.0_x64__8wekyb3d8bbwe because this package depends on a framework that could not be found. Provide the framework "Microsoft.Midi.GmDls" published by "CN=Microsoft Corporation,
O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 1.0.0.0, along with this package to install. The frameworks with name "Microsoft.Midi.GmDls" currently installed are: {}
NOTE: For additional information, look for [ActivityId] a8fe462b-9854-0000-e89d-ffa85498d801 in the Event Log or use the command line Get-AppPackageLog -ActivityID a8fe462b-9854-0000-e89d-ffa85498d801
At line:1 char:1
+ Add-AppXPackage ..\..\Downloads\Microsoft.WindowsTerminalPreview_Win1 ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\WDAGUt...bbwe.msixbundle:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand

PS C:\Users\WDAGUtilityAccount\Desktop\Sandbox>
```

Using the log command shown I get the following:

```PS C:\Users\WDAGUtilityAccount\Desktop\Sandbox> Get-AppPackageLog -ActivityID a8fe462b-9854-0000-e89d-ffa85498d801

Time                      ID           Message
----                      --           -------
16/07/2022 06:52:28       603          Started deployment Add operation on a package with main parameter Microsoft.WindowsTerminalPreview_Win10_1.15.1862.0_8wekyb3d8bbwe.msixbundle and Options 0 and 0. See http://go.microsoft.com/fwlink/?LinkId=235160
                                       for help diagnosing app deployment issues.
16/07/2022 06:52:28       164          The app bundle signature was validated for core content of the app bundle published by CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US. App packages won't be validated until they
                                       are read.
16/07/2022 06:52:28       391          The bundle streaming reader was created successfully for bundle Microsoft.WindowsTerminalPreview_3001.15.1862.0_neutral_~_8wekyb3d8bbwe.
16/07/2022 06:52:28       10002        Creating Resiliency File C:\ProgramData\Microsoft\Windows\AppRepository\132af96f-5fa3-4106-b259-56083586f40a_S-1-5-21-2047949552-857980807-821054962-504_2.rslc for Add Operation on Package
                                       Microsoft.WindowsTerminalPreview_3001.15.1862.0_neutral_~_8wekyb3d8bbwe.
16/07/2022 06:52:28       607          Deployment Add operation on package Microsoft.WindowsTerminalPreview_3001.15.1862.0_neutral_~_8wekyb3d8bbwe has been de-queued and is running for user SID S-1-5-21-2047949552-857980807-821054962-504.
16/07/2022 06:52:28       614          Bundle Uri file:///C:/Users/WDAGUtilityAccount/Downloads/Microsoft.WindowsTerminalPreview_Win10_1.15.1862.0_8wekyb3d8bbwe.msixbundle contains packages: "CascadiaPackage_1.15.1862.0_x64.msix":
                                       Microsoft.WindowsTerminalPreview_1.15.1862.0_x64__8wekyb3d8bbwe "CascadiaPackage_1.15.1862.0_x86.msix": Microsoft.WindowsTerminalPreview_1.15.1862.0_x86__8wekyb3d8bbwe "CascadiaPackage_1.15.1862.0_ARM64.msix":
                                       Microsoft.WindowsTerminalPreview_1.15.1862.0_arm64__8wekyb3d8bbwe.
16/07/2022 06:52:28       157          The app package signature was validated for core content of the app package published by CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US. Payload won't be validated until the files
                                       are read.
16/07/2022 06:52:29       216          App manifest validation warning: Declared namespace http://schemas.microsoft.com/developer/appx/2015/build is inapplicable, it will be ignored during manifest processing.
16/07/2022 06:52:29       170          The streaming reader was created successfully for app package Microsoft.WindowsTerminalPreview_1.15.1862.0_x64__8wekyb3d8bbwe.
16/07/2022 06:52:29       854          Successfully added the following uri(s) to be processed:  file:///C:/Users/WDAGUtilityAccount/Downloads/Microsoft.WindowsTerminalPreview_Win10_1.15.1862.0_8wekyb3d8bbwe.msixbundle
                                       file:///C:/Users/WDAGUtilityAccount/Downloads/Microsoft.WindowsTerminalPreview_Win10_1.15.1862.0_8wekyb3d8bbwe.msixbundle\CascadiaPackage_1.15.1862.0_x64.msix.
16/07/2022 06:52:29       629          No applicable cross-architecture framework was found for package Microsoft.WindowsTerminalPreview_1.15.1862.0_x64__8wekyb3d8bbwe. The package will be installed with matching-architecture framework only.
16/07/2022 06:52:29       628          Windows cannot install package Microsoft.WindowsTerminalPreview_1.15.1862.0_x64__8wekyb3d8bbwe because this package depends on a framework that could not be found. Provide the framework "Microsoft.Midi.GmDls"
                                       published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 1.0.0.0, along with this package to install. The
                                       frameworks with name "Microsoft.Midi.GmDls" currently installed are: {}
16/07/2022 06:52:29       605          The last successful state reached was Indexed. Failure occurred before reaching the next state Resolved. hr: 0x80073CF3
16/07/2022 06:52:29       401          Deployment Add operation with target volume C: on Package Microsoft.WindowsTerminalPreview_3001.15.1862.0_neutral_~_8wekyb3d8bbwe from:  (Microsoft.WindowsTerminalPreview_Win10_1.15.1862.0_8wekyb3d8bbwe.msixbundle)
                                        failed with error 0x80073CF3. See http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app deployment issues.
16/07/2022 06:52:29       404          AppX Deployment operation failed for package Microsoft.WindowsTerminalPreview_3001.15.1862.0_neutral_~_8wekyb3d8bbwe with error 0x80073CF3. The specific error text for this failure is: Windows cannot install
                                       package Microsoft.WindowsTerminalPreview_1.15.1862.0_x64__8wekyb3d8bbwe because this package depends on a framework that could not be found. Provide the framework "Microsoft.Midi.GmDls" published by "CN=Microsoft
                                       Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 1.0.0.0, along with this package to install. The frameworks with name
                                       "Microsoft.Midi.GmDls" currently installed are: {}


PS C:\Users\WDAGUtilityAccount\Desktop\Sandbox>
```

I did note a query about Midi dependencies in #13252 however I could not find what that dependency installer was when not developing but simply trying to install Windows Terminal Preview.

**Comment by crondrift** [comment]
Workaround: Install the "MidiVoices.appx" package from the Windows SDK (...\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.Midi.GmDls\10.0.10240.0\Appx\Neutral\MidiVoices.appx).

**Comment by glawrence** [comment]
Thanks @crondrift is that package available to download on it's own from Microsoft's site, do you know?

**Comment by crondrift** [comment]
No, I don't think so - at least I haven't found it.

**Comment by elsaco** [comment]
@glawrence if you have the Windows SDK already installed on your host copy the 'MidiVoices.appx` package to the sandbox, otherwise download the ISO SDK and extract it w/out having to install the whole SDK.

It does work. After running `Add-AppxPackage -Path .\Microsoft.WindowsTerminalPreview_Win10_1.15.1862.0_8wekyb3d8bbwe.msixbundle -DependencyPath .\Microsoft.VCLibs.x64.14.00.Desktop.appx,.\MidiVoices.appx` this is the output:



![win_sandbox_wt_preview](https://user-images.githubusercontent.com/3933920/179402308-8484d4dc-34dd-49a5-abd2-e5d8197bda47.png)


`MidiVoices` package is only 3MB, so it can be attached to an email very easy.

**Comment by glawrence** [comment]
Thanks @elsaco I have a working workaround now, hopefully a better fix will be available

**Comment by zadjii-msft** [comment]
We are tracking the removal of this dependency in #13252, which should have been fixed in #13471, and should be fixed by the time 1.15 goes to Stable. Thanks!

/dup #13252
